package.json

```diff
@@ -31,7 +33,7 @@
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/chelm/featureservice.git"
+    "url": "https://github.com/dmfenton/featureservice.git"
```

Comment:
this doesn't make a lot of sense... are we just forking featureservice since @chelm didn't give us commit access to this repo? If so the PR is unnecessary, and we should move whichever is the main repo to koopjs so we don't have to deal with something like this again.